engine/src/configuration/applier/servicegroup.cc

```diff
@@ -279,3 +447,41 @@ void applier::servicegroup::_resolve_members(
     }
   }
 }
+#else
+/**
+ * @brief Resolve the servicegroup sg_conf, so for each of its servicegroup
+ * member, we get all its service members and copy them into sg_conf.
+ * Once this done, resolved is completed with the name of sg_conf.
+ *
+ * @param s The full configuration for this poller.
+ * @param sg_conf The servicegroup configuration to resolve.
+ * @param resolved The set of servicegroup configurations already resolved.
+ * @param sg_by_name A const table of servicegroup configurations indexed by
+ * their name.
+ */
+void applier::servicegroup::_resolve_members(
+    configuration::State& s,
+    configuration::Servicegroup* sg_conf,
+    absl::flat_hash_set<std::string_view>& resolved,
+    const absl::flat_hash_map<std::string_view, configuration::Servicegroup*>&
+        sg_by_name) {
+  for (auto& sgm : sg_conf->servicegroup_members().data()) {
+    configuration::Servicegroup* sgm_conf =
+        sg_by_name.at(std::string_view(sgm));
+    if (sgm_conf == nullptr)
```

Comment:
never happen?

Reply:
Yes it can, we parse service descriptions stored in members, they are provided by the user when he declares the servicegroup. And these names can be wrong.